src/CommunityPatch/Patches/Perks/Cunning/Roguery/BriberPatch.cs

```diff
@@ -0,0 +1,177 @@
+      var villagerSurrenderHash = VillagerSurrenderTargetMethodInfo.MakeCilSignatureSha256();
+      var caravansBribeHash = CaravansBribeTargetMethodInfo.MakeCilSignatureSha256();
+      var caravansSurrenderHash = CaravansSurrenderTargetMethodInfo.MakeCilSignatureSha256();
+      return villagerBribeHash.MatchesAnySha256(VillagerBribeHashes) && 
+        villagerSurrenderHash.MatchesAnySha256(VillagerSurrenderHashes) &&
+        caravansBribeHash.MatchesAnySha256(CaravansBribeHashes) && 
+        caravansSurrenderHash.MatchesAnySha256(CaravansSurrenderHashes);
+    }
+
+    public override void Apply(Game game) {
+      var textObjStrings = TextObject.ConvertToStringList(
+        new List<TextObject> {
+          _perk.Name,
+          _perk.Description
+        }
+      );
+
+      _perk.Initialize(
+        textObjStrings[0],
+        textObjStrings[1],
+        _perk.Skill,
+        (int) _perk.RequiredSkillValue,
+        _perk.AlternativePerk,
+        _perk.PrimaryRole, 15f,
+        _perk.SecondaryRole, _perk.SecondaryBonus,
+        _perk.IncrementType
+      );
+
+      if (Applied) return;
+      CommunityPatchSubModule.Harmony.Patch(VillagerBribeTargetMethodInfo, new HarmonyMethod(VillagerBribePatchMethodInfo));
+      CommunityPatchSubModule.Harmony.Patch(VillagerSurrenderTargetMethodInfo, new HarmonyMethod(VillagerSurrenderPatchMethodInfo));
+      CommunityPatchSubModule.Harmony.Patch(CaravansBribeTargetMethodInfo, new HarmonyMethod(CaravansBribePatchMethodInfo));
+      CommunityPatchSubModule.Harmony.Patch(CaravansSurrenderTargetMethodInfo, new HarmonyMethod(CaravansSurrenderPatchMethodInfo));
+      Applied = true;
+    }
+
+    // ReSharper disable once RedundantAssignment
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    public static bool PrefixVillageBribe(ref bool __result) {
+      __result = Bribe(3, .05f, .4f);
+      return false;
+    }
+    
+    // ReSharper disable once RedundantAssignment
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    public static bool PrefixVillageSurrender(ref bool __result) {
+      __result = Bribe(4, .05f, .1f);
+      return false;
+    }
+    
+    // ReSharper disable once RedundantAssignment
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    public static bool PrefixCaravansBribe(ref bool __result) {
+      __result = Bribe(4, .1f, .6f);
+      return false;
+    }
+    
+    // ReSharper disable once RedundantAssignment
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    public static bool PrefixCaravansSurrender(ref bool __result) {
+      __result = Bribe(7, .1f, .1f);
+      return false;
+    }
+    
+    private static bool Bribe(int randomIndex, float phaseOneAcceptablePowerRatio, float phaseTwoAcceptablePowerRatio) {
+      var isLogicalSurrender = PartyBaseHelper.DoesSurrenderIsLogicalForParty(MobileParty.ConversationParty, MobileParty.MainParty, phaseOneAcceptablePowerRatio);
+      var bribeSuccessChances = isLogicalSurrender ? 67 : 33;
```

Comment:
Put some comments around this, 67 is 2/3rds rounded up, 33 is 1/3rd rounded down?

Reply:
Versus your description in the PR, note the subtle differences;
When your party is stronger than the villager/caravan in a certain proportion, you get 67% chance of success. Default is 34%. (?)
Otherwise you receive only 33%. Default is 15%. (?)

Reply:
Is there some way to grab these defaults and add perk.PrimaryBonus to them?

Reply:
Is there a way to extract these defaults somewhere?

Reply:
Looks like not really  - you have them hashed, so this is good enough.